### PR TITLE
[ko] Update outdated files in dev-1.20-ko.8 (p3)

### LINKED ISSUE #27491: Outdated files in the dev-1.20-ko.8 branch
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.20-ko.8 branch.

### 74 files to be modified 
1. [x] M1.  `content/en/docs/concepts/architecture/control-plane-node-communication.md`  | 6(+XS) 6(-)
1. [ ] M2.  `content/en/docs/concepts/architecture/nodes.md`  | 13(+S) 2(-)
1. [x] M3.  `content/en/docs/concepts/cluster-administration/addons.md`  | 1(+XS) 0(-)
1. [x] M4.  `content/en/docs/concepts/cluster-administration/manage-deployment.md`  | 2(+XS) 2(-)
1. [x] M5.  `content/en/docs/concepts/cluster-administration/proxies.md`  | 1(+XS) 1(-)
1. [ ] M6.  `content/en/docs/concepts/configuration/configmap.md`  | 2(+XS) 1(-)
1. [x] M7.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 9(+XS) 20(-)
1. [ ] M8.  `content/en/docs/concepts/configuration/secret.md`  | 9(+XS) 9(-)
1. [x] M9.  `content/en/docs/concepts/containers/container-lifecycle-hooks.md`  | 2(+XS) 1(-)
1. [x] M10.  `content/en/docs/concepts/containers/images.md`  | 1(+XS) 1(-)
1. [x] M11.  `content/en/docs/concepts/extend-kubernetes/_index.md`  | 1(+XS) 1(-)
1. [x] M12.  `content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md`  | 1(+XS) 1(-)
1. [x] M13.  `content/en/docs/concepts/extend-kubernetes/extend-cluster.md`  | 1(+XS) 1(-)
1. [x] M14.  `content/en/docs/concepts/overview/components.md`  | 2(+XS) 2(-)
1. [x] M15.  `content/en/docs/concepts/overview/working-with-objects/names.md`  | 4(+XS) 1(-)
1. [ ] M16.  `content/en/docs/concepts/overview/working-with-objects/namespaces.md`  | 2(+XS) 2(-)
1. [ ] M17.  `content/en/docs/concepts/scheduling-eviction/assign-pod-node.md`  | 9(+XS) 10(-)
1. [x] M18.  `content/en/docs/concepts/scheduling-eviction/kube-scheduler.md`  | 2(+XS) 0(-)
1. [x] M19.  `content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md`  | 7(+XS) 5(-)
1. [x] M20.  `content/en/docs/concepts/security/controlling-access.md`  | 1(+XS) 1(-)
1. [x] M21.  `content/en/docs/concepts/services-networking/connect-applications-service.md`  | 1(+XS) 1(-)
1. [ ] M22.  `content/en/docs/concepts/services-networking/ingress.md`  | 1(+XS) 1(-)
1. [ ] M23.  `content/en/docs/concepts/services-networking/service-topology.md`  | 2(+XS) 2(-)
1. [ ] M24.  `content/en/docs/concepts/services-networking/service.md`  | 15(+S) 8(-)
1. [x] M25.  `content/en/docs/concepts/storage/persistent-volumes.md`  | 1(+XS) 1(-)
1. [x] M26.  `content/en/docs/concepts/storage/storage-classes.md`  | 2(+XS) 2(-)
1. [x] M27.  `content/en/docs/concepts/storage/volume-pvc-datasource.md`  | 1(+XS) 1(-)
1. [ ] M28.  `content/en/docs/concepts/storage/volumes.md`  | 2(+XS) 2(-)
1. [x] M29.  `content/en/docs/concepts/workloads/controllers/deployment.md`  | 1(+XS) 1(-)
1. [ ] M30.  `content/en/docs/concepts/workloads/controllers/job.md`  | 1(+XS) 1(-)
1. [ ] M31.  `content/en/docs/concepts/workloads/controllers/replicaset.md`  | 2(+XS) 2(-)
1. [x] M32.  `content/en/docs/concepts/workloads/controllers/replicationcontroller.md`  | 11(+S) 11(-)
1. [x] M33.  `content/en/docs/concepts/workloads/pods/init-containers.md`  | 4(+XS) 4(-)
1. [x] M34.  `content/en/docs/concepts/workloads/pods/pod-lifecycle.md`  | 1(+XS) 1(-)
1. [x] M35.  `content/en/docs/reference/_index.md`  | 30(+M) 9(-)
1. [x] M36.  `content/en/docs/reference/access-authn-authz/authorization.md`  | 1(+XS) 1(-)
1. [ ] M37.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 1(+XS) 4(-)
1. [x] M38.  `content/en/docs/reference/glossary/cloud-controller-manager.md`  | 1(+XS) 1(-)
1. [x] M39.  `content/en/docs/reference/kubectl/cheatsheet.md`  | 4(+XS) 1(-)
1. [x] M40.  `content/en/docs/reference/kubectl/overview.md`  | 1(+XS) 1(-)
1. [ ] M41.  `content/en/docs/reference/scheduling/config.md`  | 6(+XS) 2(-)
1. [x] M42.  `content/en/docs/reference/scheduling/policies.md`  | 3(+XS) 5(-)
1. [x] M43.  `content/en/docs/setup/best-practices/cluster-large.md`  | 3(+XS) 4(-)
1. [ ] M44.  `content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md`  | 4(+XS) 4(-)
1. [x] M45.  `content/en/docs/setup/production-environment/tools/kubespray.md`  | 1(+XS) 1(-)
1. [x] M46.  `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`  | 31(+M) 22(-)
1. [x] M47.  `content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md`  | 4(+XS) 2(-)
1. [x] M48.  `content/en/docs/tasks/access-application-cluster/access-cluster.md`  | 5(+XS) 5(-)
1. [x] M49.  `content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md`  | 49(+M) 42(-)
1. [x] M50.  `content/en/docs/tasks/administer-cluster/access-cluster-services.md`  | 1(+XS) 1(-)
1. [x] M51.  `content/en/docs/tasks/administer-cluster/extended-resource-node.md`  | 1(+XS) 1(-)
1. [x] M52.  `content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md`  | 4(+XS) 2(-)
1. [x] M53.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 2(+XS) 2(-)
1. [ ] M54.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md`  | 1(+XS) 1(-)
1. [x] M55.  `content/en/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md`  | 1(+XS) 1(-)
1. [x] M56.  `content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md`  | 1(+XS) 1(-)
1. [x] M57.  `content/en/docs/tasks/configure-pod-container/static-pod.md`  | 12(+S) 16(-)
1. [x] M58.  `content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md`  | 1(+XS) 1(-)
1. [x] M59.  `content/en/docs/tasks/extend-kubernetes/setup-extension-api-server.md`  | 1(+XS) 1(-)
1. [ ] M60.  `content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md`  | 3(+XS) 4(-)
1. [ ] M61.  `content/en/docs/tasks/job/fine-parallel-processing-work-queue.md`  | 2(+XS) 2(-)
1. [x] M62.  `content/en/docs/tasks/manage-daemon/rollback-daemon-set.md`  | 1(+XS) 1(-)
1. [x] M63.  `content/en/docs/tasks/manage-daemon/update-daemon-set.md`  | 2(+XS) 2(-)
1. [x] M64.  `content/en/docs/tasks/manage-gpus/scheduling-gpus.md`  | 1(+XS) 1(-)
1. [x] M65.  `content/en/docs/tasks/run-application/delete-stateful-set.md`  | 2(+XS) 2(-)
1. [x] M66.  `content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md`  | 1(+XS) 1(-)
1. [x] M67.  `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md`  | 3(+XS) 5(-)
1. [x] M68.  `content/en/docs/tasks/tools/_index.md`  | 3(+XS) 3(-)
1. [x] M69.  `content/en/docs/tutorials/_index.md`  | 2(+XS) 0(-)
1. [x] M70.  `content/en/docs/tutorials/clusters/apparmor.md`  | 2(+XS) 2(-)
1. [x] M71.  `content/en/docs/tutorials/hello-minikube.md`  | 4(+XS) 4(-)
1. [x] M72.  `content/en/docs/tutorials/services/source-ip.md`  | 1(+XS) 1(-)
1. [x] M73.  `content/en/docs/tutorials/stateful-application/basic-stateful-set.md`  | 1(+XS) 1(-)
1. [x] M74.  `content/en/docs/tutorials/stateless-application/guestbook.md`  | 25(+S) 24(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.20-ko.8` branch.

For example:
```
# checkout `upstream/dev-1.20-ko.8`
...
# check what is updated in the upstream 
git diff upstream/dev-1.20-ko.7 upstream/dev-1.20-ko.8 -- content/en/docs/concepts/architecture/control-plane-node-communication.md
# apply changes to content/ko
vi content/ko/docs/concepts/architecture/control-plane-node-communication.md
...
# commit and push
...
# make PR to `upstream/dev-1.20-ko.8`
```

## Pages to Update

Pages in content/ko

### PR DESCRIPTION
**Pull request by jihoon-seo**
[ko] Update outdated files in dev-1.21-ko.1 (p5) (#27589) PR을 참고하였고,
제가 업데이트한 번역도 있습니다. :)

---

Resolves #27491

> ### 30 files to be modified
> 1. [ ]  M1.  `content/en/docs/concepts/architecture/control-plane-node-communication.md`  | 6(+XS) 6(-)
> 2. [ ]  M2.  `content/en/docs/concepts/architecture/nodes.md`  | 13(+S) 2(-)
> 3. [ ]  M3.  `content/en/docs/concepts/cluster-administration/addons.md`  | 1(+XS) 0(-)
> 4. [ ]  M4.  `content/en/docs/concepts/cluster-administration/manage-deployment.md`  | 2(+XS) 2(-)
> 5. [ ]  M5.  `content/en/docs/concepts/cluster-administration/proxies.md`  | 1(+XS) 1(-)
> 6. [ ]  M6.  `content/en/docs/concepts/configuration/configmap.md`  | 2(+XS) 1(-)
> 7. [ ]  M7.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 9(+XS) 20(-)
> 8. [ ]  M8.  `content/en/docs/concepts/configuration/secret.md`  | 9(+XS) 9(-)
> 9. [ ]  M9.  `content/en/docs/concepts/containers/container-lifecycle-hooks.md`  | 2(+XS) 1(-)
> 10. [ ]  M10.  `content/en/docs/concepts/containers/images.md`  | 1(+XS) 1(-)
> 11. [ ]  M11.  `content/en/docs/concepts/extend-kubernetes/_index.md`  | 1(+XS) 1(-)
> 12. [ ]  M12.  `content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md`  | 1(+XS) 1(-)
> 13. [ ]  M13.  `content/en/docs/concepts/extend-kubernetes/extend-cluster.md`  | 1(+XS) 1(-)
> 14. [ ]  M14.  `content/en/docs/concepts/overview/components.md`  | 2(+XS) 2(-)
> 15. [ ]  M15.  `content/en/docs/concepts/overview/working-with-objects/names.md`  | 4(+XS) 1(-)
> 16. [ ]  M16.  `content/en/docs/concepts/overview/working-with-objects/namespaces.md`  | 2(+XS) 2(-)
> 17. [ ]  M17.  `content/en/docs/concepts/scheduling-eviction/assign-pod-node.md`  | 9(+XS) 10(-)
> 18. [ ]  M18.  `content/en/docs/concepts/scheduling-eviction/kube-scheduler.md`  | 2(+XS) 0(-)
> 19. [ ]  M19.  `content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md`  | 7(+XS) 5(-)
> 20. [ ]  M20.  `content/en/docs/concepts/security/controlling-access.md`  | 1(+XS) 1(-)
> 21. [ ]  M21.  `content/en/docs/concepts/services-networking/connect-applications-service.md`  | 1(+XS) 1(-)
> 22. [ ]  M22.  `content/en/docs/concepts/services-networking/ingress.md`  | 1(+XS) 1(-)
> 23. [ ]  M23.  `content/en/docs/concepts/services-networking/service-topology.md`  | 2(+XS) 2(-)
> 24. [ ]  M24.  `content/en/docs/concepts/services-networking/service.md`  | 15(+S) 8(-)
> 25. [ ]  M25.  `content/en/docs/concepts/storage/persistent-volumes.md`  | 1(+XS) 1(-)
> 26. [ ]  M26.  `content/en/docs/concepts/storage/storage-classes.md`  | 2(+XS) 2(-)
> 27. [ ]  M27.  `content/en/docs/concepts/storage/volume-pvc-datasource.md`  | 1(+XS) 1(-)
> 28. [ ]  M28.  `content/en/docs/concepts/storage/volumes.md`  | 2(+XS) 2(-)
> 29. [ ]  M29.  `content/en/docs/concepts/workloads/controllers/deployment.md`  | 1(+XS) 1(-)
> 30. [ ]  M30.  `content/en/docs/concepts/workloads/controllers/job.md`  | 1(+XS) 1(-)

### CODE REVIEW

File: content/ko/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ echo $pods
 pi-5rwd7
 ```
 
-여기서 셀렉터는 잡의 셀렉터와 동일하다.  `--output=jsonpath` 옵션은 반환된 목록의
-각각의 파드에서 이름을 가져와서 표현하는 방식을 지정한다.
+여기서 셀렉터는 잡의 셀렉터와 동일하다.  `--output=jsonpath` 은
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/controllers/job.md:101]:
```suggestion
여기서 셀렉터는 잡의 셀렉터와 동일하다.  `--output=jsonpath` 는
```

- '는'이 읽기가 좋을 것 같습니다. ^^

**Review by pjhwa — COMMENTED** [review]
@jihoon-seo 양이 많았는데 고생많으셨습니다. 감사합니다!!!

2가지 의견을 드렸습니다. 확인부탁드려요~

**Comment by jihoon-seo** [comment]
@pjhwa 리뷰 감사합니다!
리뷰 의견을 반영하고 commit, squash, force-push 했습니다.
검토를 부탁드립니다. 😊

**Comment by pjhwa** [comment]
수고하셨습니다! 감사합니다.

/lgtm

**Comment by seokho-son** [comment]
감사합니다!
/approve

**Pull request merged**